roles/github/defaults/main.yml

```diff
@@ -5,7 +5,7 @@ github_environment_selector:
 
 github_kayobe_environments: []
 
-github_runs_on: [self-hosted]
+github_runs_on: [kayobe, openstack]
```

Comment:
Could you update the README?

Reply:
README has been updated with the new defaults.